src/main/scala/is/hail/utils/Py4jUtils.scala

```diff
@@ -0,0 +1,43 @@
+package is.hail.utils
+
+import scala.collection.JavaConverters._
+
+trait Py4jUtils {
+  def arrayToArrayList[T](arr: Array[T]): java.util.ArrayList[T] = {
+    val list = new java.util.ArrayList[T]()
+    for (elem <- arr)
+      list.add(elem)
+    list
+  }
+
+  def iterableToArrayList[T](it: Iterable[T]): java.util.ArrayList[T] = {
+    val list = new java.util.ArrayList[T]()
+    for (elem <- it)
+      list.add(elem)
+    list
+  }
+
+  def arrayListToISeq[T](al: java.util.ArrayList[T]): IndexedSeq[T] = al.asScala.toIndexedSeq
```

Comment:
This returns a Vector.  What about `al.toArray(new Array[T](al.length)): IndexedSeq[T]`?